src/plugins/deploy/azureDeployPlugin.ts

```diff
@@ -4,11 +4,41 @@ import { FunctionAppService } from "../../services/functionAppService";
 
 export class AzureDeployPlugin {
   public hooks: { [eventName: string]: Promise<any> };
+  public commands: any;
 
   public constructor(private serverless: Serverless, private options: Serverless.Options) {
     this.hooks = {
-      "deploy:deploy": this.deploy.bind(this)
+      "deploy:deploy": this.deploy.bind(this),
+      "deploy:list:list": this.list.bind(this),
     };
+
+    this.commands = {
+      deploy: {
+        commands: {
+          list: {
+            usage: "List deployments",
+            lifecycleEvents: [
+              "list"
+            ]
+          }
+        }
+      }
+    }
+  }
+
+  private async list() {
+    this.serverless.cli.log("Listing deployments");
+    const resourceService = new ResourceService(this.serverless, this.options);
+    const deployments = await resourceService.listDeployments();
+    let stringDeployments = "\n\nDeployments";
+    for (const dep of deployments) {
```

Comment:
If no deployments exist consider adding a line that says "No deployments found for resource group" or similar.